app/javascript/packages/document-capture/context/in-person.ts

```diff
@@ -29,6 +39,8 @@ export interface InPersonContextProps {
 }
 
 const InPersonContext = createContext<InPersonContextProps>({
+  locationsURL: '',
+  addressSearchURL: '',
```

Comment:
There aren't good default values for these. I could move the logic for calculating them to here, or follow the pattern for `inPersonURL` (which is for it to be optional and no default is defined)

Reply:
I am not sure the best answer here. I'd like to hear opinions on this.